### DISCUSSION TITLE
How do I disable auto_scroll down?

### DISCUSSION THREAD

**Discussion by MikePresman** [wezterm/wezterm#3492] [Q&A] [answered]
Sometimes when I'm scrolling up to read logs while the terminal is outputting more streams I get interrupted in my scrolling up and get thrown back down.
How can I disable this while maintaining the option to go to the bottom whenever I want with `end` or `pagedown` keys?

This is pretty annoying.

Thanks !

**Comment by wez** [comment, marked as answer]
wezterm scrolls to bottom when you type input. You can control that via:
https://wezfurlong.org/wezterm/config/lua/config/scroll_to_bottom_on_input.html

wezterm won't scroll on output, unless you have hit your scrollback limit, in which case it must evict lines and scroll.
You can avoid that by sending SIGSTOP (usually CTRL-S) to pause the program.  You can later resume it with CTRL-Q.

**Reply by MikePresman** [reply]
Amazing, thank you !
